kvbc/src/merkle_tree_db_adapter.cpp

```diff
@@ -285,7 +295,7 @@ RawBlock DBAdapter::getRawBlock(const BlockId &blockId) const {
 
   // loookup keys |NonProvable|blockid|
   // add to the keyValues
-  const auto &non_provable_key = DBKeyManipulator::genNonProvableBlockDbKey(blockId, Sliver{});
+  const auto &non_provable_key = DBKeyManipulator::genNonProvableDbKey(blockId, Sliver{});
```

Comment:
Does it make sense to name that `firstNonProvableForBlock`?

Reply:
good idea, done